packages/firestore/test/integration/api/smoke.test.ts

```diff
@@ -15,108 +15,108 @@
  * limitations under the License.
  */
 
-import * as firestore from '@firebase/firestore-types';
 import { expect } from 'chai';
 
 import { EventsAccumulator } from '../util/events_accumulator';
-import * as integrationHelpers from '../util/helpers';
-
-const apiDescribe = integrationHelpers.apiDescribe;
+import {
+  collection,
+  doc,
+  DocumentReference,
+  DocumentSnapshot,
+  getDoc,
+  getDocs,
+  onSnapshot,
+  orderBy,
+  query,
+  QuerySnapshot,
+  setDoc,
+  where
+} from '../util/firebase_export';
+import {
+  apiDescribe,
+  toDataArray,
+  withTestCollection,
+  withTestDbs,
+  withTestDoc
+} from '../util/helpers';
 
 apiDescribe('Smoke Test', (persistence: boolean) => {
   it('can write a single document', () => {
-    return integrationHelpers.withTestDoc(persistence, ref => {
-      return ref.set({
+    return withTestDoc(persistence, ref => {
+      return setDoc(ref, {
         name: 'Patryk',
         message: 'We are actually writing data!'
       });
     });
   });
 
   it('can read a written document', () => {
-    return integrationHelpers.withTestDoc(persistence, ref => {
+    return withTestDoc(persistence, ref => {
       const data = {
         name: 'Patryk',
         message: 'We are actually writing data!'
       };
-      return ref
-        .set(data)
-        .then(() => {
-          return ref.get();
-        })
-        .then((doc: firestore.DocumentSnapshot) => {
+      return setDoc(ref, data)
+        .then(() => getDoc(ref))
+        .then(doc => {
           expect(doc.data()).to.deep.equal(data);
         });
     });
   });
 
   it('can read a written document with DocumentKey', () => {
-    return integrationHelpers.withTestDoc(persistence, ref1 => {
-      const ref2 = ref1.firestore.collection('users').doc();
+    return withTestDoc(persistence, (ref1, db) => {
+      const ref2 = doc(collection(db, 'users'));
       const data = { user: ref2, message: 'We are writing data' };
-      return ref2.set({ name: 'patryk' }).then(() => {
-        return ref1
-          .set(data)
-          .then(() => {
-            return ref1.get();
-          })
-          .then((doc: firestore.DocumentSnapshot) => {
+      return setDoc(ref2, { name: 'patryk' }).then(() => {
+        return setDoc(ref1, data)
+          .then(() => getDoc(ref1))
+          .then(doc => {
             const recv = doc.data()!;
             expect(recv['message']).to.deep.equal(data.message);
             const user = recv['user'];
-            // Make sure it looks like a DocumentRef.
-            expect(user.set).to.be.an.instanceof(Function);
-            expect(user.onSnapshot).to.be.an.instanceof(Function);
-            expect(user.id).to.deep.equal(ref2.id);
+            expect(user).to.be.an.instanceof(DocumentReference);
```

Comment:
what happened to the other `expect` statements?

Reply:
I added one of them back, but the others don't apply anymore since they actually test for members of the v8 API.